src/golang/lib/repos/sqlite/artifact.go

```diff
@@ -87,7 +87,7 @@ func (*artifactReader) GetIDsByDAGAndDownstreamOPBatch(
 	// Get all the unique `artifact_id`s with an outgoing edge to an operator specified by `operatorIds`
 	// from workflow DAGs specified by `workflowDagIds`.
 	query := fmt.Sprintf(
-		`SELECT DISTINCT from_id AS artifact_id 
+		`SELECT DISTINCT from_id AS id 
```

Comment:
@saurav-c I had to change the column name to match the struct, otherwise we will get a database scan error.

Reply:
Got it thanks!